onnxruntime/core/providers/cpu/cpu_execution_provider.h

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include "core/graph/onnx_protobuf.h"
 #include "core/framework/allocatormgr.h"
```

Comment:
Is this needed given there are no other changes in this file?

Reply:
windows would have compile error on common/constants.h wo/ it
constexpr bool OPTIONAL = false;

---
In reply to: [397641170](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397641170) [](ancestors = 397641170)